pgtoolkit/log/parser.py

```diff
@@ -152,9 +185,10 @@ def stage3(self, record):
 
 
 class PrefixParser(object):
-    # PrefixParser extracts information from the beginning of each log lines,
-    # parameterized by log_line_prefix.
-    #
+    """ Extract record metadata from PostgreSQL log line prefix.
+
+    .. automethod:: from_configuration
+    """
```

Comment:
This looks unrelated to this commit. Would be nice to move this into a dedicated commit.